etc/events.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!--
+  ~ @author Aligent Consulting Team
+  ~ @copyright Copyright (c) 2022 Aligent Consulting. (http://www.aligent.com.au)
+  -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
+    <event name="sales_order_save_commit_after">
```

Comment:
This works well. But another one might be `checkout_submit_after` which only fires once after the order is placed.

This one for example might send order updates as well (comment added, status update) etc. This is perfectly valid and might even be the necessary use case. But just to note

Reply:
The intention was to send updates as well, letting middleware decide whether or not to act on them